docker-compose.el9.yml

```diff
@@ -172,7 +172,7 @@ x-rpmbuild:
         protobuf_c_min_version: *protobuf_c_min_version
     proj:
       image: rpmbuild-proj
-      version: &proj_version 9.1.0-1
+      version: &proj_version 9.1.1-1
       defines:
         data_version: "1.11"
```

Comment:
Upgrade the data version to [1.12](https://github.com/OSGeo/PROJ-data/releases/tag/1.12.0).

Reply:
Fixed in 8821c17.